platform/src/main/java/org/stellar/anchor/platform/event/KafkaSession.java

```diff
@@ -33,17 +37,33 @@
 import org.stellar.anchor.util.Log;
 
 public class KafkaSession implements EventService.Session {
-
   final KafkaConfig kafkaConfig;
   final String sessionName;
   final String topic;
   Producer<String, String> producer = null;
   Consumer<String, String> consumer = null;
+  String sslKeystoreLocation;
+  String sslTruststoreLocation;
 
-  KafkaSession(KafkaConfig kafkaConfig, String sessionName, EventQueue queue) {
+  KafkaSession(KafkaConfig kafkaConfig, String sessionName, EventQueue queue) throws IOException {
     this.kafkaConfig = kafkaConfig;
     this.sessionName = sessionName;
     this.topic = queue.name();
+
+    // If the keystore and truststore files exist, use them, otherwise, use the resources
+    if (new File(kafkaConfig.getSslKeystoreLocation()).exists()) {
```

Comment:
Is the default value of `SslKeyStoreLocation`, null? Do we need a null check here?

Reply:
Added "SASL_SSL" check. If "SASL_SSL" is set, the keystore locations must be set. 

[ANCHOR-824](https://stellarorg.atlassian.net/browse/ANCHOR-824) is created for better validation.